R/utilities-sensitivity-analysis.R

```diff
@@ -77,8 +77,23 @@ individualSensitivityAnalysis <- function(simFilePath,
                                           resultsFileName = resultsFileName) {
   # Load simulation to determine number of parameters valid for sensitivity analysis
   sim <- loadSimulation(simFilePath)
+
+  allVariableParameterPaths <- ospsuite::potentialVariableParameterPathsFor(simulation = sim)
+
+
   if (is.null(variableParameterPaths)) { # If no parameters to vary specified, vary all parameters valid for sensitivity analysis
-    variableParameterPaths <- ospsuite::potentialVariableParameterPathsFor(simulation = sim)
+    variableParameterPaths <- allVariableParameterPaths
+  } else {
+    #if a variableParameterPaths input is provided, ensure that all its elements exist within allVariableParameterPaths.  If not, give an error.
+
+    if(!all( variableParameterPaths %in% allVariableParameterPaths) ){

```

Comment:
This would fail if someone tries to vary a formula parmaeter for example that is not in the list of potentialxxx. But Ithink this is ok in that case. Thoughts @Yuri05 ?